### DISCUSSION TITLE
Why do I have to pass the first parameter of useSWRMutation. I just want to make a network request but not to mutate anything.

### DISCUSSION THREAD

**Discussion by ZhuHuaijun** [vercel/swr#2625] [Q&A] [answered]
# Bug report

## Description / Observed Behavior

What kind of issues did you encounter with SWR?

## Expected Behavior

How did you expect SWR to behave here?

## Repro Steps / Code Example

Or share your code snippet or a [CodeSandbox](https://codesandbox.io) link is also appreciated!

## Additional Context

SWR version.

**Comment by promer94** [comment, marked as answer]
The key is used for [Avoiding Race Conditions](https://swr.vercel.app/docs/mutation#avoid-race-conditions).

If you don't need this feature. You could just pass a placeholder here.
